.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "app/console"]
 	path = app/console
 	url = https://github.com/appwrite/console
-	branch = db-pools-support
+	branch = 2.2.0
```

Comment:
the 2.2.0 tag is not compatible with the db-pools branch yet. Please check with @TorstenDittmann / @ArmanNik  / @TGlide  on the status

Reply:
Posted message in PR review channel.

Reply:
@lohanidamodar you can use the `feat-cloud` branch and change the build arg (VITE_CONSOLE_MODE I think ) in the docker-compose file to `cloud`